crates/nu-command/src/math/round.rs

```diff
@@ -92,8 +102,6 @@ fn operate(value: Value, head: Span, precision: Option<i64>) -> Value {
                 span,
             },
         },
-        Value::Int { .. } => value,
-        Value::Error { .. } => value,
```

Comment:
(I currently struggle to create a passable error that doesn't immediately bubble up or gets ignored like in `do -i`)

Removing this `Value::Error` match arm alters the behavior when receiving a collection of values that contain some errors. This would now always replace them with the type error.

Unsure about the practical impact, but there was at least a historical reason why we had the separate match arm for errors coming in.

Reply:
Makes sense, adding it back then